README.md

```diff
@@ -36,8 +36,57 @@ If you find yourself with linting exceptions, to automatically resolve all issue
 
 ### How to run the tool
 
+First create the merge database:
+
+```
+CREATE DATABASE merge_db CHAR SET uf8;
+```
+
+Then load one of the dumps into `merge_db`. After that run the following script:
+```
+use merge_db;
+
+insert into person_attribute_type
+    (name, description, searchable, creator, date_created, retired, uuid) values
+    ('Source DB ID', 'ID from source database', false, 1, curdate(), false, '8d793bee-c2cc-11de-8d13-0010c6dffd23');
+```
+
+
+To execute the setup run:
+
+```
+java  -jar migrator.jar setup
+```
+
+
+To merge OpenMRS databases run:
+
+```
+java -jar migrator.jar run
+```
+
+This will run PDI transformations using `ETL_SOURCE_DATABASE` as the input database. The output will be saved in a database called `merge_db`.
```

Comment:
the name of the output db should be a setting, @kaweesi lets include this as part of ticket 100